modules/client-configuration/pages/supported-features-rh.adoc

```diff
@@ -13,59 +13,216 @@ The operating system you run on a client is supported by the organization that s
-| Service pack migration | N/A | N/A | N/A
-| Basic Virtual Guest Management icon:asterisk[role="none"] | ``Traditional`` | icon:check[role="green"] | ``Salt``
-| Advanced Virtual Guest Management icon:asterisk[role="none"] | icon:times[role="danger"] | ``Salt`` | ``Salt``
-| Virtual Guest Installation (Kickstart), as Host OS | ``Traditional`` | ``Traditional`` | icon:times[role="danger"]
-| Virtual Guest Installation (image template), as Host OS | ``Traditional`` | icon:check[role="green"] | ``Salt``
-| System deployment (PXE/Kickstart) | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
-| System redeployment (Kickstart) | ``Traditional`` | icon:check[role="green"] | ``Salt``
-| Contact methods | ``Traditional``: OSAD, RHNSD, SSH-push. ``Salt``: ZeroMQ, Salt-SSH | ``Traditional``: OSAD, RHNSD, SSH-push. ``Salt``: ZeroMQ, Salt-SSH | ``Salt``: ZeroMQ, Salt-SSH
-| Works with {productname} Proxy | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
-| Action chains | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
-| Software crash reporting | icon:times[role="danger"] | ``Traditional`` | icon:times[role="danger"]
-| Staging (pre-download of packages) | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
-| Duplicate package reporting | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
-| CVE auditing | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
-| SCAP auditing | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
-| Package verification | ``Traditional`` | ``Traditional`` | icon:times[role="danger"]
-| Package locking | ``Traditional`` | ``Traditional`` | icon:times[role="danger"]
-| System locking | ``Traditional`` | ``Traditional`` | icon:times[role="danger"]
-| System snapshot | ``Traditional`` | ``Traditional`` | icon:times[role="danger"]
-| Configuration file management | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
-| Snapshots and profiles |  ``Traditional``. ``Salt``: Profiles supported, Sync not supported | ``Traditional``. ``Salt``: Profiles supported, Sync not supported | ``Salt``: Profiles supported, Sync not supported
-| Power management |  icon:check[role="green"] | icon:check[role="green"] | ``Salt``
-| Monitoring |  ``Salt`` | ``Salt`` | ``Salt``
-| Docker buildhost |  icon:times[role="danger"] | icon:times[role="danger"] | icon:times[role="danger"]
-| Build Docker image with OS | icon:question[role="gray"] | icon:question[role="gray"] | icon:question[role="gray"]
-| Kiwi buildhost |  icon:times[role="danger"] | icon:times[role="danger"] | icon:times[role="danger"]
-| Build Kiwi image with OS |  icon:times[role="danger"] | icon:times[role="danger"] | icon:times[role="danger"]
+
+| Feature
+| {rhela}{nbsp}6
+| {rhela}{nbsp}7
+| {rhela}{nbsp}8
+
+| Client
+| {check}
+| {check}
+| {salt}
+
+| System packages
+| {redhat} / {suse} (with {es})
+| {redhat} / {suse} (with {es})
+| {redhat} / {suse} (with {es})
```

Comment:
This section is about "Native Red Hat". So I think the "with {es}" is here the wrong place. We have a special Expanded Support section.